src/Panel/panel.scss

```diff
@@ -107,11 +107,7 @@ $content-title-color: #313D4F !default;
 
     .panel-header-title {
       font-size: 16px;
-      font-weight: 700;
-    }
-
-    .panel-header-content {
-      margin-left: 5px;
```

Comment:
As discussed offline, it's hard to split pure style/color changes from structural changes. Even if we want to add or rename SASS variables, they should be adapted in projects that use them. 
IMO, if we want to keep this new style as the default for buildo components, this PR should become `breaking` and we have to evaluate/adapt every structural change in projects that use those components. 
@giogonzo, any thoughts?